pages/osx/dtrace.md

```diff
@@ -0,0 +1,38 @@
+# dtrace
+
+> Generic front-end to DTrace facility.
+> This command is a simple interface to invoke the D language compiler, retrieve buffered trace data from the DTrace kernel facility, and a set of basic routines to format and print traced data.
+> This requires root privileges.
+> More information: <https://www.unix.com/man-page/osx/1/dtrace/>.
+
+- Set dtrace's target data model which is the target architecture:
+
+`dtrace -arch {{arch_name}}`
+
+- Claim anonymous tracing state and display the traced data:
+
+`dtrace -a`
+
+- Set principal trace buffer size. Trace buffer size can include any size suffix like k, m, g or t:
+
+`dtrace -b {{buffer_size}}`
```

Comment:
```suggestion
- Set principal trace buffer size. Supported units are `k`, `m`, `g`, or `t`:

`dtrace -b {{2g}}`
```

Reply:
Is `2g` as buffer size a typo or is it a supported unit too?

Reply:
It's just an example of 2 gb.

Reply:
Done